source/community/policy/editorial.md

```diff
@@ -59,66 +61,75 @@ Specification versions prior to 1.0 should be considered experimental, are not s
 
 In addition to adhering to the guidelines above, there _MUST_ be agreement about the change among the editors. _Agreement_ is defined as follows:
 
- * There _MUST NOT_ be any "-1"s (or similar Emoji) in the latest comment from any editor before a merge.
- * __A majority__ of the editors, including the issuer of the pull request, _MUST_ actively agree ("+1") on any __non-normative__ change to a specification.
- * __All but one__ of the editors _MUST_ actively agree on any __normative__ change to a specification.
- * Even for non-specification content, __at least one__ editor, in addition to the issuer of the pull request, if applicable, must actively agree to any change on the [iiif.io][iiif-io] website.
+ * There _MUST NOT_ be any "-1"s (or similar Emoji) in the latest comment from any member of the editorial committee before a merge.
+ * __A majority__ of the editors working on the specification, including the issuer of the pull request, _MUST_ actively agree ("+1") on any __non-normative__ change to a specification.
+ * __All but one__ of the editors working on the specification _MUST_ actively agree on any __normative__ change to a specification.
 
 The editor that submitted the pull request _MUST NOT_ be the editor that merges it.
 
 In the event that an editor disagrees with a merge that has already happened, they should create a new issue that references the change in question, and explains the objection. This issue then serves as the discussion point for the objection which is subject to this same process. Reverts _SHOULD_ stay in the repository history, i.e., the process of reverting involve updating the HEAD version to reflect older content, and not erasing or re-writing history.
 
-### 2.4 Face to Face Meetings
-{: #face-to-face-meetings}
+### 2.5 Meetings
+{: #meetings}
+
+All Editors SHOULD expect to attend one editorial committee meeting per year, either in person or online, lasting 2-3 days. Every effort will be made to ensure the timing and location of the meeting is convenient for as many of the committee as possible, and it is acknowledged that not every editorial committee member will be able to attend.
+
+All Editors SHOULD expect to attend one virtual editorial committee meeting monthly to discuss the ongoing work of the community, lasting for 1 hour. 
 
-Editors will meet at least four times per year, and all editors _MUST_ attend at least three of those meetings per calendar year. Every effort will be made to co-locate at least one of the meetings with another meeting at which most of editors are already likely to be present.
+All Editors SHOULD attend as many in person and virtual meetings of TSGs and other groups for the specifications that they have agreed to work on.
```

Comment:
Remove "as many"?
Also would in-person (i.e., hyphenated) read better here?

Reply:
> All Editors SHOULD attend as many TSG meetings related to the specifications on which they are working as possible.

Reply:
I'm fine with simply removing "as many" or Jon's addition of "as possible" in a simpler sentence